geomesa-spark/geomesa-spark-sql/src/main/scala/org/locationtech/geomesa/spark/DataFrameFunctions.scala

```diff
@@ -1,14 +1,14 @@
 /***********************************************************************
- * Copyright (c) 2019 The MITRE Corporation
- * This software was produced for the U. S. Government under Basic
- * Contract No. W56KGU-18-D-0004, and is subject to the Rights in
- * Noncommercial Computer Software and Noncommercial Computer
- * Software Documentation Clause 252.227-7014 (FEB 2012).
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Apache License, Version 2.0
- * which accompanies this distribution and is available at
- * http://www.opensource.org/licenses/apache2.0.php.
- ***********************************************************************/
+* Copyright (c) 2019 The MITRE Corporation
+* This program and the accompanying materials are made available under the
+* Apache License, Version 2.0 which is available at
```

Comment:
the license check is failing b/c there needs to be an extra space in front of the first * - unfortunately it's very picky on the formatting

Reply:
Ok.  What is the recommended process to fix. Do I resubmit the PR? Do I put another commit in/on to this PR, can you change it?  Whatever needs to be done, just let me know.

Reply:
you can just push more commits to this PR, thanks